docs/layouts/partials/version-banner.html

```
@@ -0,0 +1,81 @@
+{{/*
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)